bridge/contracts/ValidatorPool.sol

```diff
@@ -466,16 +427,12 @@ contract ValidatorPool is
             uint256 payoutToken
         )
     {
-        require(
-            _validators.length() <= _maxNumValidators,
-            string(abi.encodePacked(ValidatorPoolErrorCodes.VALIDATORPOOL_MAX_VALIDATORS_MET))
-        );
-        require(
-            !_isAccusable(validator_),
-            string(
-                abi.encodePacked(ValidatorPoolErrorCodes.VALIDATORPOOL_ADDRESS_ALREADY_VALIDATOR)
-            )
-        );
+        if (_validators.length() > _maxNumValidators) {
```

Comment:
There's a bug in the current implementation, after reviwing the old code. Can you change this to be >= ?